common/util/BUILD

```diff
@@ -29,13 +29,6 @@ cc_library(
     hdrs = ["forward.h"],
 )
 
-cc_library(
-    name = "process",
```

Comment:
I think we needed process to invoke the external `diff` binary, but that is now done internally in verible, so not needed anymore.

I suggest to break 'removing process.cc' out as a separate pull request, so that we can get the codebase cleaner towards platform independence with each step.